cadastro-refeicao/Models/Item.swift

```diff
@@ -7,14 +7,30 @@
 
 import UIKit
 
-class Item: NSObject {
+class Item: NSObject, NSCoding {
+    
+    //MARK: - Atributos
     
     let nome: String
     let calorias: Double
     
+    //MARK: - init
+    
     init(nome: String, calorias: Double) {
         self.nome = nome
         self.calorias = calorias
     }
-
+    
+    //MARK: - NSCoding
+    
+    func encode(with coder: NSCoder) {
+        coder.encode(nome, forKey: "nome")
+        coder.encode(calorias, forKey: "calorias")
+    }
+    
+    required init?(coder: NSCoder) {
+        nome = coder.decodeObject(forKey: "nome") as! String
```

Comment:
Cuidado ao forçar o `unwrap`, sugestão, neste caso use uma coalescência, retornando um valor default caso o `unwrap` não seja feito, tipo isso.

``` Swift
nome = coder.decodeObject(forKey: "nome") as? String ?? "indefinido"
```